### PR TITLE
ci: publishing (tag)

### PR DESCRIPTION
**Pull request by tpluscode**

### CODE REVIEW

**Review by ludovicm67 — CHANGES_REQUESTED** [review]
There is no `release` script as I can see in the `package.json`.

Also, is tagging not good enough?
If we really want to publish on NPM, can we scope it to the `@zazuko` namespace?

**Comment by tpluscode** [comment]
> Also, is tagging not good enough?

I think changesets does not tag if there is no publishing script. Might try setting it to private to prevent NPM. Not sure about `@zazuko` org

**Review by giacomociti — APPROVED** [review]

**Comment by ludovicm67** [comment]
> I think changesets does not tag if there is no publishing script.

But if it tries to call it using `yarn release`, then it will throw an error, no?

> Might try setting it to private to prevent NPM.

Yes, good idea!
And also add the release script to `changeset tag` like I did here for example: https://github.com/zazuko/fuseki-geosparql/blob/main/package.json#L11

> Not sure about `@zazuko` org

This is only if you want to release it as a NPM module, so you can ignore this since the idea is not to publish it to NPM ;)

File: package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "build": "./build.sh",
     "trifid": "trifid --verbose --config=trifid/config.json",
     "trifid:local": "trifid --verbose --config=trifid/config.local.json",
-    "test": "standard"
+    "test": "standard",
+    "release": "changeset publish"
```
Comment (ludovicm67) [inline package.json:11]:
```suggestion
    "release": "changeset tag"
```

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -23,7 +23,11 @@ jobs:
       - name: Install Dependencies
         run: npm ci
 
-      - name: Create Release Pull Request
+      - name: Create Release Pull Request or Publish to npm
```
Comment (ludovicm67) [inline .github/workflows/release.yaml:26]:
```suggestion
      - name: Create Release Pull Request or Create a GitHub Realease + tag
```

**Comment by tpluscode** [comment]
Ok @ludovicm67, let's skip NPM for now

**Comment by ludovicm67** [comment]
Looks good!

Do you think having the `"private": true,` in the `package.json` is something that we can have?
And if so, then just update the changeset `config.json` configuration, to add the following:

```json
  "privatePackages": {
    "tag": true,
    "version": true
  },
```

like I did here: https://github.com/zazuko/fuseki-geosparql/blob/96fecb758225d1ff5232ca14fe4becf189b1ec7b/.changeset/config.json#L10C1-L13C5

**Pull request merged**